### PR TITLE
changed lambdaWater

### PR DESCRIPTION
**Pull request by icupeiro**
This solves #1141

### CODE REVIEW

**Comment by icupeiro** [comment]
@Mathadon I think the UnitTests point at changing the reference results?

File: IDEAS/Buildings/Components/InternalGains/Occupants.mo [hunk 1]
```diff
@@ -8,7 +8,7 @@ model Occupants "Constant sensible, latent and CO2 heat production per person"
   parameter Real radFra(min=0,max=1) = occupancyType.radFra
     "Radiant fraction of sensible heat exchange, default based on Ashrae fundamentals chap 18.4 for low air velocity";
 protected
-  constant Modelica.SIunits.SpecificEnthalpy lambdaWater = 2418000
+  constant Modelica.SIunits.SpecificEnthalpy lambdaWater = if (Medium.nC) >= 1 then Medium.enthalpyOfCondensingGas(T=273.15+35) else 2566120
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/InternalGains/Occupants.mo:11]: Why `Medium.nC`? Did you intend to use `Medium.nX` or `Medium.nXi`?

File: IDEAS/Buildings/Components/InternalGains/Occupants.mo [hunk 2]
```diff
@@ -85,6 +84,12 @@ Sensible heat is emitted both as convective and radiant heat using a fixed weigh
         revisions="<html>
 <ul>
 <li>
+June 3, 2020 by Iago Cupeiro:<br/>
+Changed latent heat of evaporation of water.
+This is for issue
+<a href=https://github.com/open-ideas/IDEAS/issues/635>#1141</a>.
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/InternalGains/Occupants.mo:90]: url number needs an update

File: IDEAS/Buildings/Components/InternalGains/Occupants.mo (hunk 1 above)
Comment (icupeiro) [reply IDEAS/Buildings/Components/InternalGains/Occupants.mo]: I may have copy pasted from the lines below

**Comment by Mathadon** [comment]
Looks like this needs more unit test updates.

**Comment by Mathadon** [comment]
@icupeiro LIDEAS unit tests fail. Can you see what's wrong?  https://travis-ci.com/github/open-ideas/IDEAS/jobs/347888088

**Comment by icupeiro** [comment]
@Mathadon there was a problem with the dry air model which has been corrected. Checks passed, ready for final review/merge

**Pull request merged**